package.json

```diff
@@ -80,6 +80,7 @@
     "husky": "^7.0.4",
     "js-yaml": "^4.1.0",
     "lint-staged": "^11.2.6",
+    "liquidjs": "^9.28.5",
```

Comment:
I think **liquidjs** might need to move back into the `dependencies` [vs `devDependencies`].

Reply:
+1 I’ll do that post-merge